src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFeaturedVideoSound.java

```diff
@@ -0,0 +1,70 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsJWPlayerObject;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class TestAdsFeaturedVideoSound extends TemplateNoFirstLoad {
+  private static final Duration AD_LENGTH = Duration.ofSeconds(30);
+
+  @Test(
+      groups = {"AdsFeaturedVideoSoundOasis"}
+  )
+  public void adsFeaturedVideoWithSoundOasis() {
+    verifyFeaturedVideoWithSound(AdsDataProvider.PAGE_FV_JWPLAYER_AND_SOUND.getUrl());
+  }
+  @Test(
+      groups = {"AdsFeaturedVideoSoundOasis"}
+  )
+  public void adsFeaturedVideoWithoutSoundOasis() {
+    verifyFeaturedVideoWithoutSound(AdsDataProvider.PAGE_FV_JWPLAYER_AND_SOUND.getUrl());
+  }
+
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5
+  )
+  @Test(
+      groups = {"AdsFeaturedVideoSoundMercury"}
+  )
+  public void adsFeaturedVideoWithSoundMercury() {
+    verifyFeaturedVideoWithSound(AdsDataProvider.PAGE_FV_JWPLAYER_AND_SOUND.getUrl());
+  }
+
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5
+  )
+  @Test(
+      groups = {"AdsFeaturedVideoSoundMercury"}
+  )
+  public void adsFeaturedVideoWithoutSoundMercury() {
+    verifyFeaturedVideoWithoutSound(AdsDataProvider.PAGE_FV_JWPLAYER_AND_SOUND.getUrl());
+  }
+
+  private void verifyFeaturedVideoWithSound(String pageUrl) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyFeaturedVideoWithSound" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)